packages/shorebird_code_push_protocol/lib/src/models/organization.dart

```diff
@@ -15,7 +15,11 @@ enum OrganizationRole {
   admin,
 
   /// Users who are part of the organization but have limited permissions.
-  member;
+  member,
+
+  /// Users who are not part of the organization but have visibility into it via
+  /// app collaborator permissions.
+  none;
```

Comment:
Thoughts on `limited` instead? I think none is a bit confusing since the user would have the org in their org membership but the role would be none.

Reply:
I prefer none—they don't have a role in the org. Limited raises a bunch of questions (limited how? how is limited different than member as it relates to admin)

Reply:
They technically have a role which allows them to see the org name

Reply:
Maybe external? Or guest? Are better names. Either way I’ll leave it to you

Reply:
> They technically have a role which allows them to see the org name

They don't have an organization role, they have a role as a collaborator, which grants them permission to see parts of the organization.

Reply:
Fine by me. Still feels weird to have no organization role yet still have the organization returned via the GET organization membership endpoint but we can revisit the naming later 🤷